snorkel/learning/gen_learning.py

```diff
@@ -12,11 +12,11 @@
 
```

Comment:
Can you reset this file? Looks like no functionality changes, so just want to be able to track changes correctly.